br/pkg/lightning/restore/meta_manager.go

```diff
@@ -1068,15 +1071,32 @@ func (b singleMgrBuilder) TableMetaMgr(tr *TableRestore) tableMetaMgr {
 }
 
 type singleTaskMetaMgr struct {
-	pd *pdutil.PdController
+	pd           *pdutil.PdController
+	taskID       int64
+	initialized  bool
+	sourceBytes  uint64
+	clusterAvail uint64
```

Comment:
I don't see any read from this field

Reply:
It may be read and updated by `CheckTasksExclusively`.  Although there is no place to actually read it, adding it will avoid bugs caused by future changes.